packages/react-instantsearch-hooks-web/src/ui/SearchBox.tsx

```diff
@@ -63,6 +63,7 @@ export type SearchBoxProps = Omit<
   Pick<React.ComponentProps<'form'>, 'onSubmit'> &
   Required<Pick<React.ComponentProps<'form'>, 'onReset'>> &
   Pick<React.ComponentProps<'input'>, 'placeholder' | 'onChange'> & {
+    formRef: React.RefObject<HTMLFormElement>;
```

Comment:
In this form it forces all implementations to specify `formRef`. You should make it optional to prevent that.

```suggestion
    formRef?: React.RefObject<HTMLFormElement>;
```